lib/data/external/AwsClient.js

```diff
@@ -24,6 +24,37 @@ class AwsClient {
         this._serverSideEncryption = config.serverSideEncryption;
         this._supportsVersioning = config.supportsVersioning;
         this._client = new AWS.S3(this._s3Params);
+        this._logger = new werelogs.Logger('AwsClient');
+    }
+
+    setup(cb) {
+        this._client.getBucketLocation({ Bucket: this._awsBucketName },
```

Comment:
can you add a comment saying that this call is itself redirected

Reply:
Done.